src/scripts/virtualScroll.js

```diff
@@ -1,10 +1,17 @@
 
-export default function virtualScroll(x, y, remove) {
+export default function virtualScroll(x, y, remove, options) {
   const w = x === 0 ? 0 : -1 * x;
   const h = y === 0 ? 0 : -1 * y;
 
   const translate = remove ? 'none' : `translate(${w}px,${h}px)`;
-  const html = document.documentElement;
+
+  /*
+   * value `options.scroll` indicates that `wdio-screenshot` should move
+   * an element inside DOM up instead of default `<html />`
+   */
+  let html = options && options.scroll
+    ? document.querySelector('.test-target')
+    : document.documentElement;
```

Comment:
@zinserjan we can throw an error if selector doesn't match any element in DOM to help for debugging.

Reply:
IMO the query Selector should be configuration if we are adding a scroll option.

Reply:
Looks like the forked repo ([autopilothq/wdio-screenshot](https://github.com/autopilothq/wdio-screenshot)) master branch has fixed this in a later commit. But those changes haven't been merged into the `autopilothq:set-scroll-area-bug-fix` branch, where this PR originated from.